packages/squircle-element-react/src/StaticSquircle.tsx

```diff
@@ -32,8 +40,21 @@ export const StaticSquircle = ({
       height,
       cornerRadius,
       cornerSmoothing,
+      topLeftCornerRadius,
+      topRightCornerRadius,
+      bottomRightCornerRadius,
+      bottomLeftCornerRadius,
     });
-  }, [width, height, cornerRadius, cornerSmoothing]);
+  }, [
+      width,
+      height,
+      cornerRadius,
+      cornerSmoothing,
+      topLeftCornerRadius,
+      topRightCornerRadius,
+      bottomRightCornerRadius,
+      bottomLeftCornerRadius,
+  ]);
```

Comment:
You should also add this feature to the no-js version of the squircle, which is a fallback in case js is disabled.